components/setting_picture.jsx

```diff
@@ -38,6 +38,13 @@ export default class SettingPicture extends Component {
         maxFileSize: PropTypes.number,
     };
 
+    static getDerivedStateFromProps(props, state) {
```

Comment:
Looks like this is not necessary.  Could you please check?

Reply:
Confirmed and removed in the next commit.